api/openapi/things.yml

```
@@ -169,7 +169,26 @@ paths:
           description: Missing or invalid content type.
         "500":
           $ref: "#/components/responses/ServiceError"
-
+    delete:
+      summary: Delete thing for a thing with the given id.
+      description: |
+        Delete thing removes a thing with the given id from repo
+        and removes all the policies related to this thing.
+      tags:
+        - Things
+      parameters:
+        - $ref: "#/components/parameters/ThingID"
+      security:
+        - bearerAuth: []
+      responses:
+        "204":
+          $ref: "#/components/responses/ThingDeleteRes"
```

No need to reference this since it it only description
```suggestion
          description: Thing deleted.
```

---

done